babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: ['@babel/preset-env', '@babel/preset-react'],
```

Comment:
TO DO: check why I don't need this in my package json, only existing in my package lock